### PR TITLE
Added policy replications for queues

### PR DESCRIPTION
**Pull request by obino**

### CODE REVIEW

**Comment by obino** [comment]
OCD build https://ocd.appscale.com:8080/job/Daily%20Build/3244/

**Comment by obino** [comment]
OCD migration run https://ocd.appscale.com:8080/job/Migration/1463/

File: AppController/lib/taskqueue.rb [hunk 1]
```diff
@@ -108,6 +105,12 @@ def self.start_master(clear_data, verbose)
     # First, start up RabbitMQ.
     start_rabbitmq
 
+    # The master rabbitmq will set the policy for replicating the messages
+    # on the queues.
+    cmd = "#{RABBITMQCTL} set_policy ha-all "
+    cmd += "\"\" '{\"ha-mode\":\"all\", \"ha-sync-mode\": \"automatic\"}'"
```
Comment (cdonati) [inline AppController/lib/taskqueue.rb]: I would recommend either using `%Q[string]` or building the hash in ruby and then doing a json serialization. All the escape characters makes it more difficult to read.

**Comment by obino** [comment]
OCD build https://ocd.appscale.com:8080/job/Daily%20Build/3264/

**Pull request merged**